Unable to install brewsci/homebrew-science/ipopt

Trying to install brewsci/homebrew-science/ipopt fails with this:

```
==> Downloading http://mumps.enseeiht.fr/MUMPS_5.1.1.tar.gz
Already downloaded: /Users/merahul/Library/Caches/Homebrew/mumps-5.1.1.tar.gz
==> make alllib LIBEXT=.dylib AR= -dynamiclib -Wl,-install_name -Wl,/usr/local/Cellar/mumps/5.1.1_1/lib/$(notdir $@) -undefined dynamic_lookup -o  RANLIB=echo OPTF=-O CDEFS=-DAdd_ O
Last 15 lines from /Users/merahul/Library/Logs/Homebrew/mumps/01.make:
clang -fPIC -I../include -O  -c symbfac.c -o symbfac.o
clang -fPIC -I../include -O  -c interface.c -o interface.o
clang -fPIC -I../include -O  -c sort.c -o sort.o
clang -fPIC -I../include -O  -c minpriority.c -o minpriority.o
dynamiclib -Wl,-install_name -Wl,/usr/local/Cellar/mumps/5.1.1_1/lib/libpord.dylib -undefined dynamic_lookup -o libpord.dylib graph.o gbipart.o gbisect.o ddcreate.o ddbisect.o nestdiss.o multisector.o gelim.o bucket.o tree.o symbfac.o interface.o sort.o minpriority.o 
make[2]: dynamiclib: No such file or directory
make[2]: [libpord.dylib] Error 1 (ignored)
echo libpord.dylib
libpord.dylib
if [ "./PORD/lib/" != "" ] ; then \
	  cp ./PORD/lib//libpord.dylib lib/libpord.dylib; \
	fi;
cp: ./PORD/lib//libpord.dylib: No such file or directory
make[1]: *** [lib/libpord.dylib] Error 1
make: *** [c] Error 2
```

I was asked to try using mumps from dpo/openblas as per https://github.com/brewsci/homebrew-science/issues/19. However, when I try to install ipopt, it fails with the above message while installing mumps 5.1.1.

The version of mumps available from dpo/openblas is 5.1.2.

How do I make 'brew install ipopt' use mumps from 'dpo/openblas' rather than from 'brewsci/homebrew-science'?

--- a/ipopt.rb
+++ b/ipopt.rb
@@ -1,0 +1,47 @@
+class Ipopt < Formula
+  desc "Large-scale nonlinear optimization package"
+  homepage "https://projects.coin-or.org/Ipopt"
+  url "https://www.coin-or.org/download/source/Ipopt/Ipopt-3.12.9.tgz"
+  sha256 "8ff3fe1a8560896fc5559839a87c2530cac4ed231b0806e487bfd3cf2d294ab8"
+  head "https://projects.coin-or.org/svn/Ipopt/trunk", :using => :svn
+
+  option "without-test", "Skip build-time tests (not recommended)"
+
+  depends_on "ampl-mp"
+  depends_on "gcc"
+  # IPOPT is not able to use parallel MUMPS.
+  depends_on "mumps" => "without-mpi"
+  depends_on "openblas"
+  depends_on "pkg-config" => :build
+
+  def install
+    mumps_libs = %w[-ldmumps -lmumps_common -lpord -lmpiseq]
+    mumps_incdir = Formula["mumps"].opt_libexec/"include"
+    mumps_libcmd = "-L#{Formula["mumps"].opt_lib} " + mumps_libs.join(" ")
+
+    args = ["--disable-debug",
+            "--disable-dependency-tracking",
+            "--prefix=#{prefix}",
+            "--with-mumps-incdir=#{mumps_incdir}",
+            "--with-mumps-lib=#{mumps_libcmd}",
+            "--enable-shared",
+            "--enable-static",
+            "--with-blas-incdir=#{Formula["openblas"].opt_include}",
+            "--with-blas-lib=-L#{Formula["openblas"].opt_lib} -lopenblas",
+            "--with-lapack-incdir=#{Formula["openblas"].opt_include}",
+            "--with-lapack-lib=-L#{Formula["openblas"].opt_lib} -lopenblas",
+            "--with-asl-incdir=#{Formula["ampl-mp"].opt_include}/asl",
+            "--with-asl-lib=-L#{Formula["ampl-mp"].opt_lib} -lasl"]
+
+    system "./configure", *args
+    system "make"
+    ENV.deparallelize # Needs a serialized install
+    system "make", "test" if build.with? "test"
+    system "make", "install"
+  end
+
+  test do
+    # IPOPT still fails to converge on the Waechter-Biegler problem?!?!
+    system "#{bin}/ipopt", "#{Formula["ampl-mp"].opt_pkgshare}/example/wb"
+  end
+end

